libs/datalayout/build.gradle

```diff
@@ -10,10 +9,18 @@ ext {
     sparkVersion = '3.1.1'
     springVersion = '2.7.8'
     hadoopVersion = '2.10.0'
+    sparkVersionSuffix = "3.1"
+    openhouseSparkRuntimeModule = ":integrations:spark:spark-${sparkVersionSuffix}:openhouse-spark-runtime_2.12"
+    tablesTestFixturesModule = ":tables-test-fixtures:tables-test-fixtures_2.12"
```

Comment:
are changes in this file relevant to this PR?

Reply:
yes, in this PR i must unpin 1.2 from the convention plugin removed above. i do this by removing convention plugin, and defining explicitly here and without any strictly version constraints. 

this is so that consumers of this library (:apps) can override with their required version such as iceberg 1.5.2